lib/spack/spack/verify.py

```diff
@@ -110,6 +110,8 @@ def check_file_manifest(filename):
     dirname = os.path.dirname(filename)
 
     results = VerificationResults()
+    # What should be the correct exit condition as sys root seems wrong to me?
+    # There is a failing assert in spack\spack\test\verification.py(line 230) due to this
```

Comment:
@scheibelp I made an iterative approach using pathlib.Path(...).parents but it had the same error as the original. Though, this program works for other tests that use the function. I looked deeper into the specific test and found the reason its failing on Windows is that the test is assuming there is no instances of .spack as it goes through each parent directory of the given directory file path. If this was the case it would return the expected error message. However, on my system there is a .spack dir in my name directory. This causes the program to check if that .spack directory has a install_manifest.json. It doesn't so it raises the "missing manifest" error. I'm wondering if this is just something on my personal computer that is resulting in the error and if it would just pass on develop. Also, there are two possible ways for the program to get the expected error message. I'm assuming that the program wants to return from the error message within the loop rather than the error message just pass the try exception logic.